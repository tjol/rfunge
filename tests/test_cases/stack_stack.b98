a81p12{..11g.v>'!,@^g18<
>0}11g.#*}#;}{^v{1876; }
^.g-10-50p21b< >0{3u2k.^
     @,!'u@#-10..}.<