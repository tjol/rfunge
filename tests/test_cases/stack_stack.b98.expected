1 0 10 10 10 8 15 1 10 7 6 